### ISSUE TITLE
Default image to show if element is empty

### ISSUE THREAD

**Issue by VernitaJ** [marcopeg-dev/m07-cv-frontend#48]
When profile_pic is empty on backend, default image should be shown.

What is looks like now:
![image](https://user-images.githubusercontent.com/55649851/76316959-a8f1cc00-62db-11ea-9b89-0edba1ac237a.png)

**Comment by NicolaWinterson** [comment]
Hi, the PR above seems to fix this issue. If you agree you can close this issue.

**Comment by AHMADanis** [comment]
@VernitaJ, I don't think this is a good idea to have a transparent placeholder in absence of user's image. If the user don't have an image then there should be something to represent the user.

**Comment by VernitaJ** [comment]
@AHMADanis 
1. It's not working correctly after merge.
2. The image thumbnail is still there,  which is why we should apply a default image.  Default image can be something like 
![image](https://user-images.githubusercontent.com/55649851/76406584-41df2080-638a-11ea-93be-45cdc72aaa96.png)
